src/base/utils/password.h

```diff
@@ -28,11 +28,13 @@
 
 #pragma once
 
-class QByteArray;
+#include  <QByteArray>
+
 class QString;
 
 namespace Utils::Password
 {
+    inline const QByteArray defaultPassword = QByteArrayLiteral("ARQ77eY1NUZaQsuDHbIMCA==:0WMRkYTUWVT9wVvdDtHAjU9b3b7uB8NR1Gur2hmQCvCDpm39Q+PsJRJPaCU51dEiz+dTzh8qbPsL8WkFljQYFQ==");
```

Comment:
FYI, `QByteArray` can't be `constexpr`.

Reply:
I don't think `constexpr` is important in this case.

Reply:
no need to obsess with constexpr when it is already compile-time:
https://doc.qt.io/qt-6/qbytearray.html#QByteArrayLiteral
>The macro generates the data for a [QByteArray](https://doc.qt.io/qt-6/qbytearray.html) out of the string literal ba at compile time. Creating a [QByteArray](https://doc.qt.io/qt-6/qbytearray.html) from it is free in this case, and the generated byte array data is stored in the read-only segment of the compiled object file.